Source compilation error: ./cabal.project: openBinaryFile: does not exist (No such file or directory)

I have tried to build SimpleX from source using Docker (as advised in CLI.md).
I run:
`sudo DOCKER_BUILDKIT=1 docker build --output ~/.local/bin .`

Here's the error I get:

```
#22 933.6 [66 of 73] Compiling Simplex.Chat     ( src/Simplex/Chat.hs, dist/build/Simplex/Chat.o, dist/build/Simplex/Chat.dyn_o )
#22 933.6 
#22 933.6 src/Simplex/Chat.hs:1317:76: error:
#22 933.6     * Exception when trying to run compile-time code:
#22 933.6         ./cabal.project: openBinaryFile: does not exist (No such file or directory)
#22 933.6       Code: simplexmqCommitQ
#22 933.6     * In the untyped splice: $(simplexmqCommitQ)
#22 933.6      |
#22 933.6 1317 |   ShowVersion -> pure $ CRVersionInfo $ coreVersionInfo $(buildTimestampQ) $(simplexmqCommitQ)
#22 933.6      |                                                                            ^^^^^^^^^^^^^^^^^^^
#22 933.6 cabal: Failed to build simplex-chat-4.5.1.0 (which is required by
#22 933.6 exe:simplex-chat from simplex-chat-4.5.1.0, exe:simplex-bot-advanced from
#22 933.6 simplex-chat-4.5.1.0 and others). See the build log above for details.
#22 933.6 
------
executor failed running [/bin/sh -c cabal install]: exit code: 1
```                                                                             

Any ideas on how to resolve this issue?

--- a/simplex-chat.cabal
+++ b/simplex-chat.cabal
@@ -16,6 +16,7 @@ license-file:   LICENSE
 build-type:     Simple
 extra-source-files:
     README.md
+    cabal.project
 
 flag swift
   description: Enable swift JSON format

